packages/config/config/devices/0x010f/templates/fgs-2xx.json

```
@@ -0,0 +1,302 @@
+// Template for FGS-211, FGS-212, FGS-221 and FGS-222
+{
+	"manufacturer": "Fibargroup",
+	"manufacturerId": "0x010f",
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Switch 1",
+			"maxNodes": 16
+		},
+		"2": {
+			"label": "Switch 2",
+			"maxNodes": 16
+		},
+		"3": {
+			"label": "Controller Updates",
+			"description": "Reports state of devices.",
```

```suggestion
			"label": "Lifeline",
```
We should stay in Z-Wave slang :) Please check the other association definitions too.